packages/block-library/src/editor.scss

```diff
@@ -64,13 +63,6 @@
 	@include gradient-colors-deprecated();
 }
 
-// This CSS Custom Properties aren't used anymore as defaults,
-// but we still need to keep them for backward compatibility.
-:where(.editor-styles-wrapper) {
-	--wp--preset--font-size--normal: 16px;
-	--wp--preset--font-size--huge: 42px;
-}
-
```

Comment:
Could we also delete in this file the following rules that are duplicated in common.scss?

```css
:root .editor-styles-wrapper {
	@include background-colors-deprecated();
	@include foreground-colors-deprecated();
	@include gradient-colors-deprecated();
}

:where(.editor-styles-wrapper) .has-normal-font-size {
	font-size: var(--wp--preset--font-size--normal);
}

:where(.editor-styles-wrapper) .has-huge-font-size {
	font-size: var(--wp--preset--font-size--huge);
}
```

?

They are loaded in the top-level and editor frames.

Reply:
> Could we also delete in this file the following rules that are duplicated in [common.scss](https://github.com/WordPress/gutenberg/blob/965454f2e166819d7c78b1770340411643a24d48/packages/block-library/src/common.scss#L7)?

From what I've tested, removing these styles should be fine. As I remember, @ellatrix was involved in the process of injecting the document stylesheet into the iframe editor instance. Maybe she has some knowledge.

Reply:
Thanks again. I'll remove them and ask for forgiveness later :trollface:

Reply:
I have no idea. If both stylesheets are present in the iframe, it should be fine

Reply:
Thanks for checking!